.changes/unreleased/Fixes-20230918-145127.yaml

```
@@ -0,0 +1,6 @@
+kind: Fixes
+body: Add column quotes in macro to avoid build failure
+time: 2023-09-18T14:51:27.988997-06:00
+custom:
+  Author: seanglynn-thrive
+  Issue: "6154"
```

Did you mean #8667 instead of #6154?

```suggestion
  Issue: "8667"
```